hedera-node/src/main/java/com/hedera/services/state/logic/BlockManager.java

```diff
@@ -84,12 +88,65 @@ public long getCurrentBlockNumber() {
 		return networkCtx.get().getManagedBlockNo();
 	}
 
+	public long getProvisionalBlockNumber() {
+		return provisionalBlockNumber;
+	}
+
+	public org.hyperledger.besu.datatypes.Hash getProvisionalBlockHash(long blockNumber) {
+		if (blockNumber == provisionalBlockNumber) {
```

Comment:
By definition, this must return `0x` because the `provisionalBlockNumber` is the current block, and the hash of the current block is never available.